### DISCUSSION TITLE
Visual bug on Handlebars Formula?

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2938] [Help]
Hi!

After updated to 0.9.161, formula field doesn't save correctly when has bracket.

1. Create a formula column;
2. Input a formula usign bracket;
3. Click on Confirm;
4. Formula looks wrong (only visual). **Do nothing**. Click on Save Columns;
5. Edit column;
6. Highlight formula text;
7. Click on Save Columns.

Evidence:

https://user-images.githubusercontent.com/53205779/136602767-8a9a9f1c-7feb-4172-b7f2-b8d5c2329a2c.mp4

**Comment by mjashanks** [comment]
Thanks for reporting! I logged this as an issue: https://github.com/Budibase/budibase/issues/2955
